src/server/services/controllers/accountController.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+var models = require('../../db/schema');
+
+function sendError(errorCode, res, err, message) {
+    res.status(errorCode).json({
```

Comment:
We should probably do a little null-checking on `err` - if `err` is `null` or `undefined`, then `err.message` below will throw a reference exception.